views/css/templates/index.php

```diff
@@ -1,35 +0,0 @@
-<?php
-/*
-* 2007-2019 PrestaShop
```

Comment:
We need to keep this index.php file, it's a protection we put for server without a proper .htaccess config

Reply:
@matks I remove the index.php, because I remove the directory :)